shared/src/main/java/eu/darkbot/shared/utils/PortalJumper.java

```diff
@@ -13,20 +15,29 @@ public class PortalJumper {
     protected final MovementAPI movement;
     protected final GroupAPI group;
 
+    protected WindowAPI window;
```

Comment:
This should be final and initialized in all codepaths, change the preexisting constructor to have this in. I'm aware it's a breaking change but that's what the alternative plugin-api constructor is for (for 3rd parties).